app/utils/participant_attributes_util.js

```diff
@@ -0,0 +1,24 @@
+import realmHelper from '../helpers/realm_helper';
+import { getAttributesByColumns } from '../helpers/scorecard_attributes_helper';
+
+const participantAttributesUtil = (() => {
+  return {
+    parse
+  };
+
+  function parse(scorecard) {
+    let participants = realmHelper.queryAsJSON('Participant', scorecard);
```

Comment:
@limkimsan please consider to remove realmHelper and write a raw code to be like below:

let participants = JSON.parse(JSON.stringify(Participant.getAll(`scorecard_uuid='${scorecard.uuid}'`))

please note that every method that touch realm object database should be model class

Reply:
Noted bong, I will update them.